src/PlotDocs.jl

```diff
@@ -211,7 +211,7 @@ const ATTRIBUTE_TEXTS = Dict(
     :Series => "These attributes apply to individual series (lines, scatters, heatmaps, etc)",
     :Plot => "These attributes apply to the full Plot. (A Plot contains a tree-like layout of Subplots)",
     :Subplot => "These attributes apply to settings for individual Subplots.",
-    :Axis => "These attributes apply to an individual Axis in a Subplot (for example the `subplot[:xaxis]`)",
+    :Axis => "These attributes apply by default to all Axes in a Subplot (for example the `subplot[:xaxis]`). You can also specific the x, y, or z axis for each of these attributes by prefixing the attribute name with x, y, or z (for example `xmirror` only sets the mirror attribute for the x axis)",
```

Comment:
```suggestion
    :Axis => """
    These attributes apply by default to all Axes in a Subplot (for example the `subplot[:xaxis]`).
    !!! info
        You can also specific the x, y, or z axis for each of these attributes by prefixing the attribute name with x, y, or z
        (for example `xmirror` only sets the mirror attribute for the x axis).
    """,
```
Sorry it took so long. I wanted to make sure the docs build, which required a few small PRs on Plots.
How about putting the text in an info box? Here's a preview (with a different text I had on a local branch):
![Screenshot_20201014_213608](https://user-images.githubusercontent.com/16589944/96038436-6989f000-0e67-11eb-9eee-1e7a22b6653b.png)

Reply:
This is perfect!